index.html

```diff
@@ -33,8 +35,146 @@ <h1>I’m Mehdi jdir. <br>
+            <div class="presentation">
+                <p class="project_title">Multi-Post
+                    Stories</p>
+                <ul class="frame">
+                    <li class="client">CANOPY</li>
+                    <li class="dot"></li>
+                    <li class="list">Back End Dev</li>
+                    <li class="dot"></li>
+                    <li class="list">2015</li>
+                </ul>
+                <p class="primary">A daily selection of privately personalized reads;
+                 no accounts or sign-ups required.</p>
+                <ul class="tag">
+                    <li class="bouton b1">HTML</li>
+                    <li class="bouton b2">CSS</li>
+                    <li class="bouton b3">JavaScript</li>
+                </ul>
+                <button class="btn"><p class="btn_txt">See project</p></button>
+            </div>
+        </section>
+
+        <section class="works">
+            <img class="image" src="./img/image3.png" alt="image3">
+            <div class="presentation">
+                <p class="project_title">Tonic</p>
+                <ul class="frame">
+                    <li class="client">CANOPY</li>
+                    <li class="dot"></li>
+                    <li class="list">Back End Dev</li>
+                    <li class="dot"></li>
+                    <li class="list">2015</li>
+                </ul>
+                <p class="primary">A daily selection of privately personalized reads;
+                 no accounts or sign-ups required.</p>
+                <ul class="tag">
+                    <li class="bouton b1">HTML</li>
+                    <li class="bouton b2">CSS</li>
+                    <li class="bouton b3">JavaScript</li>
+                </ul>
+                <button class="btn"><p class="btn_txt">See project</p></button>
+            </div>
+        </section>
+
+        <section class="works">
+            <img class="image" src="./img/image4.png" alt="image4">
+            <div class="presentation">
+                <p class="project_title">Multi-Post
+                    Stories</p>
+                <ul class="frame">
+                    <li class="client">CANOPY</li>
+                    <li class="dot"></li>
+                    <li class="list">Back End Dev</li>
+                    <li class="dot"></li>
+                    <li class="list">2015</li>
+                </ul>
+                <p class="primary">A daily selection of privately personalized reads;
+                 no accounts or sign-ups required.</p>
+                <ul class="tag">
+                    <li class="bouton b1">HTML</li>
+                    <li class="bouton b2">CSS</li>
+                    <li class="bouton b3">JavaScript</li>
+                </ul>
+                <button class="btn"><p class="btn_txt">See project</p></button>
+            </div>
+        </section>
+    </section>
+
+        <section class="About">
```

Comment:
- [ ] Please note that the design has a curved top right corner. Kindly implement this. 
- [OPTIONAL] I suggest that you open the design and compare it with your implementation side by side to find out which parts you have not matched to the design yet. 

![des1](https://github.com/medijdir/mobileportfolio/assets/99656687/3757d72a-5219-4313-8b87-d1e4d9be7a4c)